app/src/main/java/com/mealsmadeeasy/ui/home/suggestions/SuggestionsAdapter.kt

```diff
@@ -35,14 +40,16 @@ class SuggestionViewHolder(view: View) : RecyclerView.ViewHolder(view) {
 
     private val thumbnailView = view.findViewById<ImageView>(R.id.suggestion_thumbnail)
     private val nameView = view.findViewById<TextView>(R.id.suggestion_name)
+    private lateinit var meal: Meal
 
     init {
         view.setOnClickListener {
-            Toast.makeText(itemView.context, "TODO", Toast.LENGTH_SHORT).show()
+            suggestionListener.suggestionClicked(view, meal.id)
```

Comment:
To call the function when you use the fancy syntax, this'll become `suggestionListener(view, meal.id)`.